bin/start_lexical.sh

```diff
@@ -19,7 +18,7 @@ case "$VERSION_MANAGER" in
         asdf env erl exec "$@" &
         ;;
     rtx)
-        rtx env -s bash erl exec "$@" &
+        eval "$(rtx env -s bash erl)"
```

Comment:
```suggestion
        eval "$(rtx env -s bash)"
```
`rtx env`'s argument serves to override a given tool with a specific version. `erl` does nothing in this case